ui/package.json

```diff
@@ -3,18 +3,18 @@
     "version": "2.1.5",
     "private": true,
     "dependencies": {
+        "@vitejs/plugin-react": "^4.2.1",
         "bootstrap": "^5.3.2",
         "react": "^18.2.0",
-        "react-dom": "^18.2.0"
-    },
-    "devDependencies": {
-        "react-scripts": "^5.0.1"
+        "react-dom": "^18.2.0",
+        "vite": "^5.1.1",
```

Comment:
nit: 5.1.2?

Reply:
5.1.2 came out after I submitted my PR, I can test and let you know 😊